scripts/coordinate.js

```diff
@@ -363,6 +363,12 @@ FriendlyChat.prototype.getSession = function() {
         let renderWhenInformation = data.val().whenDescription;
         let renderWhereInformation = data.val().whereDescription;
         let renderWhoInformation = data.val().whoDescription; // To-Do: Update with first names dynamically.
+        let renderWhoAmIInformation = data.val().whoAmIDescription;
```

Comment:
Just as an FYI @omarcodex  started to removed all the let declarations in his branch to make IE11 😃